modules/controller/rpc-controller.js

```diff
@@ -798,8 +824,15 @@ class RpcController {
         res.status(202).send({
             handler_id: handlerId,
         });
-        const fileContent = req.files.file.data;
-        const fileExtension = path.extname(req.files.file.name).toLowerCase();
+        let fileContent,fileExtension;
+        if (req.files) {
```

Comment:
what if we have neither file nor data? I think that we will have an error here, maybe we can return check:
```js
if ((!req.files || !req.files.file || path.extname(req.files.file.name).toLowerCase() !== '.json') && (!req.body.data)) {
            return next({code: 400, message: 'No data provided. It is required to have assertion file or data in body, they must be in JSON-LD format.'});
```